js/apps/account-ui/src/api/methods.ts

```diff
@@ -109,8 +109,14 @@ export async function getCredentials({ signal, context }: CallOptions) {
   return parseResponse<CredentialContainer[]>(response);
 }
 
-export async function getLinkedAccounts({ signal, context }: CallOptions) {
-  const response = await request("/linked-accounts", context, { signal });
+export async function getLinkedAccounts(
+  { signal, context }: CallOptions,
+  requestParams: Record<string, string>,
+) {
```

Comment:
Let's add an explicit type here with the parameters that are allowed. Additionally, let's keep the primitive types here rather than using `string` everywhere. This avoids additional parsing using `parseInt()` or `Boolean()` in the React components.

```suggestion
export type GetLinkedAccountParams = PaginationParams & {
  linked: boolean;
};

export async function getLinkedAccounts(
  { signal, context }: CallOptions,
  requestParams: GetLinkedAccountParams,
) {
```